docs/custom-capo.md

```diff
@@ -0,0 +1,58 @@
+# Creating a custom cluster api provider ovirt and adding it to the cluster
+
+When we develop the CAPO its usually helpful to test it against a running cluster.
+The following doc explains how to do it.
+
+## Replacing the docker image
+
+1.Lunch a cluster:
+
+Start an Openshift cluster on top of oVirt as usual with the required release image.
+
+2. Create the docker image:
+
+** replace ${username} and ${tag} with the correct values
+```bash
+podman login quay.io
```

Comment:
I would recommend use $ in front of commands, specially this shows podman doesn't require root as others solutions like docker.

```
$ podman login quay.io
$ podman build -t quay.io/${username}/cluster-api-provider-ovirt:${tag} .
$ podman push quay.io/${username}/cluster-api-provider-ovirt:${tag}
```